.github/workflows/pr-preview.yml

```
@@ -0,0 +1,34 @@
+name: PR-Preview
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  REACT_APP_NETWORK_IDS: "100,10200"
+  GITHUB_ENV: "./"
```

```suggestion
  PUBLIC_URL: "./"
```
And I would still document this if possible because I don't know which command uses this and why.